README.rst

```diff
@@ -13,7 +13,11 @@ In this README document:
 Workshop Overview
 -----
 
-...
+Some tips to get the most out of the workshop:
+- make sure you run all the cells that contain code- cells further down may need the results from an earlier cell!
+- if you are confused about what a cell is doing, make sure you read the text above and below it
+- we use `this format` inside of a block of text to show that this is code that you may end up running yourself later
+- if you get stuck, do ask people seated around you and ask our coaches for help! There are no stupid questions, and your neighbors are your friends who may also learn from helping you! Along the way, you may also want to figure out what key terms to search for, where to search for it, and how to debug the problem- this will let you develop good intuition for how to search for a solution yourself next time.
```

Comment:
```suggestion
- if you get stuck, do ask people seated around you and ask our coaches for help! There are no stupid questions, and your neighbors are your friends who may also learn from helping you! Along the way, you may also want to figure out what key terms to search for, where to search for it, and how to debug the problem - this will let you develop good intuition for how to search for a solution yourself next time.
```